src/scripts/activateThebelab.js

```diff
@@ -68,6 +68,19 @@ const activateThebelab = (config, detectLanguage = true) => {
 
     loadScript('https://unpkg.com/thebelab@0.5.1/lib/index.js')
       .then(() => {
+        thebelab.on('status', (e, data) => {
+          const statusElement = document.getElementsByClassName('thebe-status-field')[0];
+          if (statusElement !== undefined) {
+            statusElement.innerHTML = `Code Cell: ${data.status.toUpperCase()}`;
+            statusElement.setAttribute('class', `thebe-status-field thebe-status-${data.status}`);
+
+            if (data.status === 'ready') {
```

Comment:
I am okay with letting the status symbol remain there indefinitely. We have a timeout of 1 hour, but if someone opens their libretexts page and runs a cell, goes afk, and then comes back, there is no indication to them that the kernel is dead. Removing the timeout entirely is what I suggest but we can discuss.

Reply:
How would removing the timeout work? There has to be a timeout somewhere, otherwise pods will just build up and our cluster will grind to a halt cuz lack of resources. We can probably play around with the timeout, though. (Of course, we're ignoring the possibility of someone DoSing us and spawn a ton of pods, but to be honest that has always been an issue and we're just dealing with it manually with Grafana alerts now. Anyway.)

Reply:
The timeout here (which I kind of miscommented on) makes the status button disappear after 3 seconds. I was suggesting removing the status button timeout, not the pod timeout.

Reply:
Ah OK, that makes sense, ignore me

Reply:
After doing some test and reading the source code, after kernel is ready, it is up to an internal package to deal with events and will not notify us in the current implementation. That is, even is the kernel is dead, the current implementation will not what that and will still show that it is `ready`. So I think keeping this is better to not confuse people that it is still alive.